scripts/ci/test-examples.sh

```diff
@@ -32,12 +32,14 @@ build_local() {
   ninja
 }
 
+export CELER_LOG=debug CELER_LOG_LOCAL=debug
+
 cd "${CELER_SOURCE_DIR}/example/minimal"
 build_local
 ./minimal
 
 
-if [ -z "CELER_DISABLE_ACCEL_EXAMPLES" ]; then
+if [ -z "${CELER_DISABLE_ACCEL_EXAMPLES}" ]; then
```

Comment:
Maybe the original writer meant
```suggestion
if [ -v CELER_DISABLE_ACCEL_EXAMPLES ]; then
```
(true is the variable is set/exist)

Reply:
I definitely just forgot to dereference, but I did't know that `-v` existed. Is that a standard POSIX `test` function? I use `-z` so that you can `export FOO=` to unset a variable, since it's easier to `export FOO=` rather than `env -u FOO` (and more intuitive to understand a false/empty variable has the same behavior as "not set"... but then again this means `FOO=0` is "true").

If we use `-v` then how should I change
https://github.com/celeritas-project/celeritas/blob/6ec7f84e9a407aae378624937c37850a603ca161/.github/workflows/build-spack.yml#L232-L237
?

Reply:
> Is that a standard POSIX test function?

It appears that it is not available everywhere (eg. MacOS).

> If we use -v then how should I change

This is a good case, not to use `-v`